### ISSUE TITLE
Markdown in Rust doc comments support

### ISSUE THREAD

**Issue by maxwase** [izhangzhihao/intellij-rainbow-brackets#483]
## Your programming language(s)

Rust

## Expected Behavior

The plugin should not colorize letters in key words and doc comments `/`

## Current Behavior
The plugin changes the color of the letters in key words and doc comments `/`   (see pictures down below)

![image](https://user-images.githubusercontent.com/23321756/92714397-d55ad380-f364-11ea-8aed-d386dce283af.png)

![image](https://user-images.githubusercontent.com/23321756/92714631-210d7d00-f365-11ea-9dfc-3bb7ebd10b20.png)

## Possible Solution

All brackets list? 

## Code snippet for reproduce (for bugs)

https://doc.rust-lang.org/core/option/enum.Option.html

## Your Environment

* Plugin version: 6.10
```
CLion 2020.2.1
Build #CL-202.6948.80, built on August 25, 2020
Runtime version: 11.0.8+10-b944.31 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.7.15-100.fc31.x86_64
GC: ParNew, ConcurrentMarkSweep
Memory: 1987M
Cores: 12
Registry: run.processes.with.pty=TRUE, documentation.show.toolbar=true, ide.allow.merge.buttons=false, search.everywhere.settings=true, debugger.watches.in.variables=false, cidr.max.intellisense.file.length=600000
Non-Bundled Plugins: name.kropp.intellij.makefile, org.jetbrains.plugins.localization, org.toml.lang, tanvd.grazi, org.rust.lang, rs.pest, izhangzhihao.rainbow.brackets
Current Desktop: KDE
```

**Comment by izhangzhihao** [comment]
But, it IS bracket, right?

**Comment by maxwase** [comment]
> But, it IS bracket, right?

`f` in the `fn` and `l` in the `let` keywords are not.
Documentary comments ± if it is, then it is not always work correctly

**Comment by izhangzhihao** [comment]
![image](https://user-images.githubusercontent.com/12044174/92929477-2ce76500-f473-11ea-93fb-5cc2e5f28e3e.png)

Ths PSI tree of this code

```
//! ```
//! let result = divide(2.0, 3.0);
//! ```
```

is 

```
FILE(0,50)
  PsiComment(<INNER_EOL_DOC_COMMENT>)('//! ```\n//! let result = divide(2.0, 3.0);\n//! ```')(0,50)
``` 

That's ... too bad, there properly nothing I could do, even after hours of debugging...

**Comment by izhangzhihao** [comment]
Please follow: https://github.com/intellij-rust/intellij-rust/pull/6455

**Comment by izhangzhihao** [comment]
[This issue already fixed in the nightly build of the Rust plugin start on Jun 11](https://github.com/intellij-rust/intellij-rust/pull/6455#issuecomment-859343401)

**Comment by izhangzhihao** [comment]
If you found your highlighting is missing, please follow: https://github.com/intellij-rust/intellij-rust/issues/6100

**Comment by maxwase** [comment]
Fixed in 0.4.149.3939-211 Rust plugin and 6.18 rainbow-brackets plugin
